TreeTracker/UI/Views/Authentication/SignIn/SignIn.storyboard

```diff
@@ -82,7 +82,7 @@
                                         <constraints>
                                             <constraint firstAttribute="height" constant="50" id="WGV-Zd-ZAY"/>
                                         </constraints>
-                                        <state key="normal" title="LOGIN">
+                                        <state key="normal" title="LOG IN">
```

Comment:
@ClarissaVinciguerra I don't think this will fix the issue. We source strings from the Localizable.strings file so you will need to change it in there.

Whilst we only have one language in the iOS app currently we intend to support more so it's important we think about localization as we build this out. Appreciate we could localise storyboard strings if we wanted but i think its best we keep all copy in one place.